pages.fr/common/at.md

```diff
@@ -4,13 +4,21 @@
 > Le service atd (ou atrun) doit être actif pour l'exécution des commandes planifiées.
 > Plus d'informations : <https://manned.org/at>.
 
+- Démarre le démon (tâche de fond) `atd` :
```

Comment:
Maybe use the word "service" instead of "démon"?
```suggestion
- Démarre `atd` comme un service (en tâche de fond) :
```

It would be coherent with the description:
> Le service atd (ou atrun) doit être actif pour l'exécution des commandes planifiées.

Source: [this French Ubuntu Documentation](https://doc.ubuntu-fr.org/tutoriel/comment_transformer_un_programme_en_service)

Quote in French: 
> Comment transformer un programme en service

> Service est traduit en anglais par daemon. Originellement DAEMON pour « Disk And Execution MONitor » est un programme réalisant des tâches de fond du système sous Unix. Appelé aussi driver sur d'autres systèmes. En temps normal, son fonctionnement ne doit pas être remarqué par l'utilisateur. Un démon peut aussi être un sous-programme appelé par un programme principal (dans ce cas on parle « sérieusement » de bibliothèque partagée).